UserInterface.cs

```diff
@@ -0,0 +1,887 @@
+﻿using System.Text;
+using AirportTicketBooking.CSVFiles;
+
+namespace AirportTicketBooking
+{
+    public class UI
+    {
+        private static char? _option;
+        private static Passenger passenger;
+        private static List<Flight> availableFlights = new();
+        private static List<Booking> passengerBookings = new();
+
+        public static void Main()
```

Comment:
for this too long filem for now i want u to move filteration logic to the services, and do the required refactoring so i can visit again and review